README.md

```diff
@@ -14,29 +8,6 @@ This repository includes files with plain SQL that can be used to recreate a dat
 
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
-
 <!-- TABLE OF CONTENTS -->
```

Comment:
- [ ] Kindly remove the`Live Demo` section from your read me file as the link you have provided is not valid.
![image](https://github.com/tsheporamantso/vet_clinic/assets/81932478/f6b9ee39-99fa-4dff-a3ee-a1bb7694c9fe)

Reply:
Removed as well

Reply:
well resolved! ✔️